spec/api_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'spec_helper'
+
+# This will run tests about the API.
+describe 'NY Movies API' do
+  before(:all) do
+   @api_data = NYMovies.new.ny_api_data
+   @api_data.retrieve_api('gladiator',ENV['API_KEY'])
+  end
+  context 'Testing Data from different outputs of the API' do
+    it 'Testing whether or not the date returns a String' do
```

Comment:
the test description need to be in the format of:-

it 'should return date as a string' do

Use the should keyword to make the description readable